tests/ckeditor/ckeditor.js

```diff
@@ -48,6 +48,14 @@ describe( 'create', function() {
 		} );
 	} );
 
+	it( 'should set configurations on the new editor', function() {
+		var CKEDITOR = modules.ckeditor;
+
+		return CKEDITOR.create( content, { test: 1 } ).then( function( editor ) {
+			expect( editor.config.test ).to.equals( 1 );
```

Comment:
Man you really can't resist using `to.equals`, huh? My inner grammar nazi cries every time I read this :crying_cat_face:
Can we agree that every expect should produce a grammatically correct sentence?
For example: *"Expect A to equal B."*, *"Expect A to be an instance of B"* etc.